docs/source/examples/simulate_RFB_example.ipynb

```diff
@@ -362,9 +362,9 @@
    "source": [
     "**In summary**, the general simulation procedure is as follows: \n",
     "\n",
-    "1. Describe flow cell with the `ZeroDModel` class.\n",
-    "2. Select the electrochemical cycling protocol class e.g., `ConstantCurrent`.\n",
-    "3. Simulate the cell for some duration by calling the `.run()` function of the cycling protocol class.\n"
+    "1. Describe flow cell with the [`ZeroDModel`](https://ericfell-rfbzero.readthedocs.io/en/latest/flowcell.html#src.rfbzero.redox_flow_cell.ZeroDModel) class.\n",
+    "2. Select the electrochemical cycling protocol class e.g., [`ConstantCurrent`](https://ericfell-rfbzero.readthedocs.io/en/latest/experiment.html#src.rfbzero.experiment.ConstantCurrent).\n",
```

Comment:
If we're going to have a link for ConstantCurrent here in the summary, it might make sense to just mention ConstantVoltage and ConstantCurrentConstantVoltage here for completeness as well

Reply:
agree, switched to: 
Select the electrochemical cycling protocol class i.e., `ConstantCurrent`, `ConstantCurrentConstantVoltage`, or `ConstantVoltage`.